src/components/video/Video.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+// import { select } from '@storybook/addon-knobs'
+import logo from '../../assets/logo.png'
+import { StyledFigure, PlaceholderButton } from './Video.styles'
+import Video from './Video'
+
+const vidId = 'liJVSwOiiwg?autoplay=1'
+const coverImg = logo
+const captionStr = 'Pride in London presents: You! Me! Us! We!'
```

Comment:
You could pop the `videoId` and `caption` into a `text` function, which means the user can dynamically modify the fields. Take a look at the `Button` story for an example.